### ISSUE TITLE
Content Feedback - New Tools Page

### ISSUE THREAD

**Issue by digitalchild** [WordPress/Learn#1974]
I see time and time again people asking about the tools creators are using for screen captures etc. I think it would be great if we had a page we could direct people to that would list the various tools people use. 

There are plenty of open source/free options for creating content. 

We could also have a section on programming related tools as well. 

## Overview
- Content title:  Learn Content Tools
- Content URL:  none yet
- Are you reporting an error (such as outdated information), or providing improvement ideas?: 

## Feedback Details

Create a new page with a list of various tools creators can use. 

## Next steps for Training Team member
Please follow the team handbook "[Validating Content Feedback issues](https://make.wordpress.org/training/handbook/training-team-how-to-guides/how-we-use-github/validating-and-applying-content-feedback/#validating-content-feedback-issues)" to validate this feedback.

**Comment by kaitohm** [comment]
This makes sense. Listing open source softwares, and even sharing step-by-step resources on how to use those would be great.

I'm thinking this content would be a handbook page, rather than something that lives on Learn. Would you agree?

Related resource: https://make.wordpress.org/training/handbook/tutorials/creating-a-tutorial/#step-1-choose-software (What we currently list in the handbook regarding recording software.)

**Comment by jonathanbossenger** [comment]
> I'm thinking this content would be a handbook page, rather than something that lives on Learn. Would you agree?

@bsanevans as someone who was part of this conversation, that was the original idea. As we don't have a "Handbook Feedback" issue template, I suggested Jamie use "Content Feedback". Maybe adding a new template for handbook updates would also be a good idea going forward. 

That being said, I also wonder if a tutorial on the different options would be useful, especially showing how to use free and open source products, instead of those with free trials that have restrictions (ie the Descript watermark).

**Comment by digitalchild** [comment]
Yes, a handbook page is where it should live. Having the tutorials might become dated quickly though due to how quickly some products change.

**Comment by kaitohm** [comment]
Cool, thanks for the clarification. We'll consider what the best template may be for the handbook as part of the GitHub project 📝 

I agree, a tutorial sounds great, but I also see they could get outdated just as quick... Maybe periodic Online Workshops? More food for thought when we discuss this in the team meeting today 🙂
